maquetado/index.html

```diff
@@ -6,15 +6,125 @@
   <title>HTML & CSS Training</title>
   <meta name="description" content="training">
   <meta name="author" content="Wolox">
-  <link rel="stylesheet"  type="text/css" href="application.css">
+  <link rel="stylesheet" type="text/css" href="application.css">
 </head>
 <body>
-  <h1 class="title">Front-End Tools</h1>
-  <div class="tech">
-    <img class="tech-logo" src="images/react_icon.svg" alt="React's logo">
-    <h3 class="tech-title">ReactJS</h3>
-    <h4 class="tech-release-date">Release: 2013</h4>
-    <p class="tech-description">React is a declarative, component-based JavaScript library for building user interfaces.</p>
+  <div class='nav'>
```

Comment:
```suggestion
  <nav class="nav">
```
For keeping the correct syntax